### ISSUE TITLE
setup.py develop fails on py36, numpy 1.16, linux

### ISSUE THREAD

**Issue by barentine** [python-microscopy/pymecompress#2]
quite possibly not an issue on numpy 1.14

```
~/code/pymecompress$ python setup.py develop
 Traceback (most recent call last):
  File "setup.py", line 28, in <module>
    conf = configuration(top_path='').todict()
  File "setup.py", line 17, in configuration
    config.add_subpackage('pymecompress')
  File "/home/smeagol/anaconda2/envs/py36/lib/python3.6/site-packages/numpy/distutils/misc_util.py", line 1036, in add_subpackage
    caller_level = 2)
  File "/home/smeagol/anaconda2/envs/py36/lib/python3.6/site-packages/numpy/distutils/misc_util.py", line 1005, in get_subpackage
    caller_level = caller_level + 1)
  File "/home/smeagol/anaconda2/envs/py36/lib/python3.6/site-packages/numpy/distutils/misc_util.py", line 942, in _get_configuration_from_setup_py
    config = setup_module.configuration(*args)
  File "./pymecompress/setup.py", line 156, in configuration
    extra_link_args=linkArgs)
NameError: name 'linkArgs' is not defined
```

**Comment by barentine** [comment]
Broken on linux. will submit PR shortly

**Comment by barentine** [comment]
Should be fixed with merge of PR #3
